sentry-samples/sentry-samples-servlet/README.md

```diff
@@ -4,7 +4,7 @@ Sample application showing how to use Sentry with a non-Spring Java servlet appl
 
 ## How to build?
 
-From the project directory execute command:
+From the module directory execute a command:
 
 ```
 ../../gradlew build
```

Comment:
should be `run` here too?

Reply:
With servlets project has to be built first and then deployed to application server. I could use some third party plugin to deploy to embedded tomcat but this would bring another dependency that I'd like to avoid.